src/azure-cli/azure/cli/command_modules/hdinsight/custom.py

```diff
@@ -316,6 +316,12 @@ def create_cluster(cmd, client, cluster_name, resource_group_name, cluster_type,
         msi_resource_id=assign_identity
     )
 
+    if encryption_at_host:
+        if disk_encryption_properties:
+            disk_encryption_properties.encryption_at_host = encryption_at_host
+        else:
+            disk_encryption_properties = DiskEncryptionProperties(encryption_at_host=encryption_at_host)
```

Comment:
I don't see the usage of `disk_encryption_properties`. Is it correct?

Reply:
> I don't see the usage of `disk_encryption_properties`. Is it correct?

It is used at line 360.